test/system/group_offer_categories_test.rb

```diff
@@ -3,11 +3,11 @@
 class GroupOfferCategoriesTest < ApplicationSystemTestCase
   def setup
     @user = create :user
-    login_as @user
   end
 
 
-  test 'create new group offer category and update it' do
+  test 'superadmin can create new group offer' do
```

Comment:
Since superadmin and department manager have the same rights, both tests could be for the superadmin.